xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -627,6 +633,7 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 		}
 
 	}
+	printf("init succeeded\n");
```

Comment:
please revert this, there is not information in it because the device id and the backend is missing. Better remove this.

Reply:
NOTE the newlines this is PART OF THE SAME OUTPUT LINE sir.